packages/types/src/definitions/EndAction.type.ts

```diff
@@ -0,0 +1,6 @@
+export enum EndAction {
+  Continue = 'continue',
+  Pause = 'pause',
+  Stop = 'stop',
+  Next = 'next'
+}
```

Comment:
enums FTY 😄

Reply:
what do you think of the implementation in the EventEditor? (dont know how to reference the code in this comment )

Reply:
For me it sounds a little complicated.

The brain needs to work a bit to read it, even though it is straightforward 

I wonder 
- are there easier ways to write that?
- is the code abstraction worth for such a simple case?

As long as it works. I will leave it up to you